src/commands/up.rs

```diff
@@ -105,6 +106,21 @@ pub struct UpOpts {
         conflicts_with = DISABLE_WASMTIME_CACHE,
     )]
     pub cache: Option<PathBuf>,
+
+    /// Print output for given component(s) to stdout/stderr
+    #[structopt(
+        name = FOLLOW_LOG_OPT,
+        long = "follow",
+        multiple = true,
+        )]
+    pub follow_components: Vec<String>,
+
+    /// Print all component output to stdout/stderr
+    #[structopt(
+        long = "follow-all",
+        conflicts_with = FOLLOW_LOG_OPT,
+        )]
+    pub follow_all_components: bool,
```

Comment:
Do we want an environment variable for this?

Reply:
We've tended to use environment variables for things that are "this is what my setup looks like and it will never change."  But if someone is running Spin repeatedly trying to track down an issue, then they may want to leave this on for the duration of that session; but maybe that's what shell history is for.  On the whole I worry that an EV could lead to surprises when you accidentally spewed logs during a demo or when you did `--follow` for one component and it conflicted with the presence of the EV.  But it's no effort to add one if people reckon it would be good!

Reply:
Ah, good point, I agree that it could be confusing.
Happy to defer this conversation to a later time (say to when we have a global configuration file).

Thanks!